### PR TITLE
Reorganize and reprioritize things in the README.

### PR DESCRIPTION
**Pull request by crazy2be**
Coming back to this project after a few years, the things in the README that
seem important are quite different than those things that we had previously
prioritized. Change the ordering, and move some of the more complicated web
setup notes into a seperate README, since most people probably won't have to
run them.

### CODE REVIEW

**Comment by crazy2be** [comment]
It's a bit nicer to just look at the new readme rather than the diff: https://github.com/crazy2be/buildblast/blob/826b64dc7038afdec2d020c7fcd222f2e357da33/README.md

File: README.md [hunk 1]
```diff
@@ -6,106 +6,20 @@ BuildBlast
...
-- **container**: Just a div in the html, we use this to hook up the actual rendering. There are some hardcoded elements in this, which we hook up to to give THREE.js a place to render (amoung other things).
-- **Conn**: A wrapper for a WebSocket, used to communicate to the server.
-- **Scene**: A THREE.Scene, this holds all the meshes and models which THREE.js will render.
-- **ChunkManager**: Manages the chunks (of blocks), and handles adding and removing them from the THREE.Scene.
-- **EntityManager**: Manages the entities (players for now, in the future hopefully more) and add/removes them from the THREE.Scene.
-- **World**: Holds both the ChunkManager and EntityManager and provides useful logic to interface with them. Can directly expose blocks but also provides most of the helper functions you should need.
-- **PlayerUI**: Handles displaying the interface elements and rendering the scene. Shouldn't hold any game data, or be required for the game to run.
+To run the www server, enable authentication, and enable multiple worlds, see [WEBREADME.md]
```
Comment (kieve) [inline README.md:22]: This link seems to be broken in the preview. But, that might just be from looking at the blob, and not viewing it as part of the GitHub pages

Comment (crazy2be) [reply README.md:22]: The link was broken, you were correct. I just sort of assumed that would work without actually checking.

Comment (crazy2be) [reply README.md:22]: I'm not sure why I expected that to work. [Clearly I shouldn't have](https://daringfireball.net/projects/markdown/syntax#link). I think I was misapplying mediawiki syntax to Mardown.

Comment (crazy2be) [reply README.md:22]: Fixed in https://github.com/crazy2be/buildblast/commit/75ff7da95459d6315adc661d027f1b3fb1f8647f

**Pull request merged**